access/GITLAB.md

```diff
@@ -0,0 +1,9 @@
+# GitLab Access Plugin
+
+We partially developed a GitLab Access Plugin, but there was little-to-no
+usage or demand for it.  Thus we decided to stop paying the maintenance cost
+of the plugin.  If you want to recover the work, find the plugin at:
+
+981ccb982c1e2b09ff98f370019748d783e050d5
```

Comment:
I'll update this to include instructions for reverting this commit after it merges.  I won't know the final hash for `git revert` until that point.